src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -1986,10 +1985,6 @@ void updateZoneSizingEndZoneSizingCalc1(EnergyPlusData &state, int const zoneNum
     }
 
     int numSpaces = 0; // Track this for averages later
-    bool heatDDNumAllSame = true;
-    bool coolDDNumAllSame = true;
-    int priorHeatDDNum = 0;
-    int priorCoolDDNum = 0;
```

Comment:
[src/EnergyPlus/ZoneEquipmentManager.cc:1989]:(style),[unreadVariable],Variable 'heatDDNumAllSame' is assigned a value that is never used.
[src/EnergyPlus/ZoneEquipmentManager.cc:1990]:(style),[unreadVariable],Variable 'coolDDNumAllSame' is assigned a value that is never used.
[src/EnergyPlus/ZoneEquipmentManager.cc:1991]:(style),[unreadVariable],Variable 'priorHeatDDNum' is assigned a value that is never used.
[src/EnergyPlus/ZoneEquipmentManager.cc:1992]:(style),[unreadVariable],Variable 'priorCoolDDNum' is assigned a value that is never used.

Reply:
Hmm, oh these were left over from plan A, but changed to plan B. Ok.